.travis.yml

```diff
@@ -93,6 +93,10 @@ jobs:
       script:
         - yarn ember test -c testem.travis-browsers.js
 
+    - name: Octane
+      env:
+      - OVERRIDE_FEATURES=EMBER_METAL_TRACKED_PROPERTIES,EMBER_GLIMMER_ANGLE_BRACKET_BUILT_INS,EMBER_GLIMMER_ANGLE_BRACKET_NESTED_LOOKUP,EMBER_NATIVE_DECORATOR_SUPPORT
+
```

Comment:
This shouldn't be necessary: we already have a build that enables _all_ the optional features, it's just in the second "stage"

Reply:
I realized this later.